pam_aad/docker/ubuntu/Dockerfile

```
@@ -0,0 +1,32 @@
+FROM ubuntu:18.04
```

Rename and bump, e.g., `pam_aad/ubuntu/Dockerfile`

```suggestion
FROM ubuntu:20.04
```